### PR TITLE
Issue 1856: Upgrade our awaitility dependencies from 3.1.2 to 4.0.3

### PR DESCRIPTION
**Pull request by kosuruvarunkumar**
Addressing the Awaitility dependency upgrade.

Fixes strongbox/strongbox#1856.

### CODE REVIEW

**Comment by carlspring** [comment]
Hi!

Thank you for your contribution!

Would you mind signing the ICLA, as described in the [Contributing](https://github.com/strongbox/strongbox/blob/master/CONTRIBUTING.md) page?

Also, please, feel free to join our [chat channel](https://chat.carlspring.org/), if you'd like to learn more about the project and/or like to find out what else you could help with.

Kind regards,

Martin

**Comment by kosuruvarunkumar** [comment]
Thanks, @carlspring for the info, I mailed the signed ICLA

**Comment by kosuruvarunkumar** [comment]
@carlspring as you acknowledged that you received the ICLA, can we have the label removed.
Thanks

File: pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>org.carlspring.strongbox</groupId>
     <artifactId>strongbox-parent</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0-issue-1856-SNAPSHOT</version>
```
Comment (carlspring) [inline pom.xml:9]:
```suggestion
    <version>1.0-SNAPSHOT</version>
```
Reverting to `1.0-SNAPSHOT` after successful testing.

File: pom.xml [hunk 2]
```diff
@@ -586,8 +586,8 @@
                 <artifactId>commons-io</artifactId>
                 <version>${version.apache.commons.io}</version>
             </dependency>
-            <!-- Override for transitive dependencies which bring older vulnerable 
-                versions of the commons-beanutils:commons-beanutils in order to not be exposed 
+            <!-- Override for transitive dependencies which bring older vulnerable
+                versions of the commons-beanutils:commons-beanutils in order to not be exposed
                 to: http://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2014-0114 -->
```
Comment (carlspring) [inline pom.xml:591]:
```suggestion
            <!-- Override for transitive dependencies which bring older vulnerable
                 versions of the commons-beanutils:commons-beanutils in order to not be exposed
                 to: http://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2014-0114 -->
```

File: pom.xml [hunk 3]
```diff
@@ -620,7 +620,7 @@
                         <artifactId>jsr311-api</artifactId>
                     </exclusion>
                     <exclusion>
-                        <!-- Excluding this transitive dependency in order 
+                        <!-- Excluding this transitive dependency in order
                             to not be exposed to: http://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2016-3720 -->
```
Comment (carlspring) [inline pom.xml:624]:
```suggestion
                        <!-- Excluding this transitive dependency in order
                             to not be exposed to: http://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2016-3720 -->
```

**Comment by carlspring** [comment]
Thank you! Congratulations on your first pull request! :)

**Pull request merged**